### PR TITLE
PrettierLintBear.py: Add more tests

### PR DESCRIPTION
**Pull request by anctartica**
Closes: https://github.com/coala/coala-bears/issues/2290

### CODE REVIEW

File: tests/js/PrettierLintBearTest.py [hunk 1]
```diff
@@ -14,6 +14,40 @@
...
+"""
+
+good_quotes = """const a = () => "Foo bar";
+
+function b(object, key) {
+  return object["key"];
+}"""
+
+bad_quotes = """const a = () => 'Foo bar';"""
+
```
Comment (MacBox7) [inline tests/js/PrettierLintBearTest.py]: Remove the extra newline present in line 28 and 35. To make your code pretty :smile:

File: tests/js/PrettierLintBearTest.py [hunk 2]
```diff
@@ -14,6 +14,38 @@
...
+good_indent = """if (x) {
+  x = 0;
+}"""
+
 PrettierLintBear = verify_local_bear(PrettierLintBear,
-                                     valid_files=(good_file,),
-                                     invalid_files=(bad_file,))
+                                     valid_files=(good_quotes,
+                                                  good_file,
+                                                  good_paranethesis,
```
Comment (li-boxuan) [inline tests/js/PrettierLintBearTest.py]: 'good_paranethesis' is wrong spelling. (same issue for 'bad_paranethesis')

Comment (anctartica) [reply tests/js/PrettierLintBearTest.py]: thanks!

**Comment by MacBox7** [comment]
Do you wish to add more tests? Or, Is it ready for final review?

**Comment by anctartica** [comment]
I think I have added adequate tests as of now. But there were other issues with PrettierLintBear when I checked it #2365. @MacBox7

**Comment by Makman2** [comment]
ack b97087cd40d5515ff75289afbef8f923ce1e467e

**Comment by Makman2** [comment]
@gitmate-bot ff

**Pull request merged**